README.markdown

```diff
@@ -31,7 +31,8 @@ This should give you the flexibility you need for customizing your input/output
 
 ### Building Your Phar
 
-Simply run `php package.php`, which will create the file: `build/doctrine-migrations.phar` for you.
+The PHAR is ready for [download here](http://github.com/downloads/ericclemmons/migrations/doctrine-migrations.phar), but if you still would like to build your PHAR
```

Comment:
@bizmate It's worth asking for the need of PHAR first, before putting so much effort to generating it. 

Usually it's integrated with another package:

- Symfony - https://github.com/doctrine/DoctrineMigrationsBundle
- Nette - https://github.com/Zenify/DoctrineMigrations
- Laravel - https://github.com/atrauzzi/laravel-doctrine
...